packages/ui/src/modal-side/ModalSide.css

```diff
@@ -38,13 +38,7 @@
   animation: modal-slide-out 300ms reverse ease-in;
 }
 
-.ui-ModalSide--Header {
-  z-index: 1;
-  position: fixed;
-  top: 0;
-}
-
-.ui-ModalSide--Header .ui-ButtonModalClose.--icon {
+.ui-ModalSide--Modal .ui-ButtonModalClose.--icon {
```

Comment:
We still need the styles from the header here as not adding a z-index makes the button go behind the images in 9.10.14.

<img width="1031" alt="Screenshot 2024-07-26 at 10 10 07 AM" src="https://github.com/user-attachments/assets/17ef0443-38a5-4df1-831e-701eaff302db">